pkgs/development/python-modules/pyasn/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, }:
+
+buildPythonPackage rec {
+  pname = "pyasn";
+  version = "1.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-6UK1SRY2Pse4tw6urs0OtOQe8bz0ojl7KabXFfzN+SU=";
+  };
+
+  doCheck = false; # Tests require internet connection which wont work
+
```

```suggestion

pythonImportsCheck = [ "pyasn" ];

```

At least do a simple import test.